### PR TITLE
UN-726: Record Revealed `gallery_heading`

### PR DESCRIPTION
**Pull request by jamesbiggs**
Ticket URL: https://national-archives.atlassian.net/browse/UN-726

## About these changes

Added gallery_heading CharField

Moved gallery_heading and gallery_images into a MultiFieldPanel to bunch them together

Added HTML for FE to pick up and style

## How to check these changes

Edit a Record Revealed page and add a "gallery heading". It should appear above the gallery (as per design on the ticket)

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer.
- [x] Checked PR title starts with ticket number as per project conventions to help us keep track of changes.
- [x] Ensured that PR includes only commits relevant to the ticket.
- [x] Waited for all CI jobs to pass before requesting a review.
- [x] Added/updated tests and documentation where relevant.

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

### CODE REVIEW

**Comment by TNA-Allan** [comment]
Hi @jamesbiggs 
image
The gallery text does not appear in the centre above Image 1 of 5 as per the ticket.

File: etna/articles/models.py [hunk 1]
```diff
@@ -371,12 +379,19 @@ class Meta:
                 FieldPanel("custom_warning_text"),
             ],
         ),
-        InlinePanel(
-            "gallery_images",
+        MultiFieldPanel(
```
Comment (TNA-Allan) [inline etna/articles/models.py:382]: Why is this changed from InlinePanel to MultiFieldPanel? Is this anywhere in the requirements?

Comment (jamesbiggs) [reply etna/articles/models.py:382]: Hi Allan, it's not part of the requirements but just how we tend to lay things out in the CMS. I've chosen to do this as it allows the "Image Gallery" components to be grouped together nicely in the CMS 🙂

**Comment by jamesbiggs** [comment]
> Hi @jamesbiggs image The gallery text does not appear in the centre above Image 1 of 5 as per the ticket.

Hi Allan, the ticket isn't for FE work - that is to be taken up by a FE dev 🙂

File: etna/articles/models.py (hunk 1 above)
Comment (TNA-Allan) [reply etna/articles/models.py:382]: Can you please make a mention in the ticket what this change is about, so that we can proceed.
_Re:  Ensured that PR includes only commits relevant to the ticket._

Comment (TNA-Allan) [reply etna/articles/models.py:382]: And how to test this change as well.

Comment (jamesbiggs) [reply etna/articles/models.py:382]: This is relevant to the ticket, it's part of the "process" I suppose 🙂 It just bunches the fields together so that they are included in a drop down - otherwise they are separate. Makes for a nicer editor experience, but is relevant as it's part of the adding of the `gallery_heading`

**Review by TNA-Allan — APPROVED** [review]
Thanks @jamesbiggs
